lib/kaffe/producer.ex

```diff
@@ -83,6 +97,11 @@ defmodule Kaffe.Producer do
   ## internal
   ## -------------------------------------------------------------------------
 
+  defp produce_list(topic, partition, message_list) when is_list(message_list) do
+    Logger.debug "event#produce_list topic=#{topic} partition=#{partition}"
+    @kafka.produce_sync(client_name(), topic, partition, "ignored", message_list)
```

Comment:
Is there any value in having this function utilize `produce_list_to_topic` (as the existing `produce_list` does) so there aren't two independent paths to the brod client invocation? I realize there might be some finagling required to support a consistent format between producing to a static partition vs. producing to a dynamically chosen partition, but it seems like we should minimize the branching in that last mile logic.

Thoughts?